v2/fleet-local/control/zk-exhibitor@.service

```diff
@@ -25,6 +25,7 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
   -e S3_BUCKET=`etcdctl get /zookeeper/s3_exhibitor_bucket` \
   -e S3_PREFIX=`etcdctl get /zookeeper/s3_exhibitor_prefix` \
   -e ZK_ENSEMBLE_SIZE=`etcdctl get /zookeeper/ensemble_size` \
+  -e ZK_PASSWORD=`etcdctl get /zookeeper/ZK_PASSWORD` \
```

Comment:
Try `etcdctl set zookeeper/ZK_PASSWORD ''`

From: Anshu Pande
Reply-To: adobe-platform/mesos-systemd
Date: Wednesday, March 2, 2016 at 9:35 AM
To: adobe-platform/mesos-systemd
Cc: Israel Derdik
Subject: Re: [mesos-systemd] testing zk authentication (#146)


In v2/fleet-local/control/zk-exhibitor@.service<https://github.com/adobe-platform/mesos-systemd/pull/146#discussion_r54729893>:

> @@ -25,6 +25,7 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
>    -e S3_BUCKET=`etcdctl get /zookeeper/s3_exhibitor_bucket` \
>    -e S3_PREFIX=`etcdctl get /zookeeper/s3_exhibitor_prefix` \
>    -e ZK_ENSEMBLE_SIZE=`etcdctl get /zookeeper/ensemble_size` \
> +  -e ZK_PASSWORD=`etcdctl get /zookeeper/ZK_PASSWORD` \


It does not allow to set it to blank even from CLI:

anshu@ip-10-43-1-211 sandbox-anshu-stack-us-east-1a-control ~ $ etcdctl set zookeeper/ZK_PASSWORD

—
Reply to this email directly or view it on GitHub<https://github.com/adobe-platform/mesos-systemd/pull/146/files#r54729893>.